src/main/kotlin/pt/up/fe/ni/website/backend/model/Activity.kt

```diff
@@ -21,6 +26,10 @@ abstract class Activity(
     @field:Size(min = Constants.Description.minSize, max = Constants.Description.maxSize)
     open val description: String,
 
+    @JoinColumn
+    @OneToMany(cascade = [CascadeType.ALL], fetch = FetchType.EAGER)
+    open var associatedRoles: List<@Valid PerActivityRole>,
```

Comment:
This has nothing to do with this PR but Activities' attributes shouldn't need to be open, can you change it?
Also, isn't val ok here?

Reply:
I can change it, no worries 
Regarding the use of var, I made it that way since I needed it for the tests and because it follows what we've done on other models (most, if not all, properties are var).